src/main/webapp/WEB-INF/redirects.properties

```diff
@@ -27,6 +27,7 @@
 /blog/2019/03/01/microprofile-concurrency.html=/blog/2019/08/16/microprofile-context-propagation.html
 /blog/2019/05/24/testing-database-connections-REST-APIs.html=/blog/2019/09/13/testing-database-connections-REST-APIs.html
 /blog/2019/09/13/microprofile-reactive-messsaging-19009.html=/blog/2019/09/13/microprofile-reactive-messaging-19009.html
+/blog/2019/11/07/ibm-red-hat-support-open-liberty.html=https://openliberty.io/blog/?search=OpenShift
```

Comment:
missed this in the original review, but I believe this needs to be 
`/blog/2019/11/07/ibm-red-hat-support-open-liberty.html=/blog/?search=OpenShift`